core/tests/protractor_desktop/additionalEditorFeatures.js

```diff
@@ -108,8 +107,8 @@ describe('Full exploration editor', function() {
   it('should handle discarding changes, navigation, deleting states, ' +
       'changing the first state, displaying content, deleting responses and ' +
       'switching to preview mode', async function() {
-    await users.createUser('user5@editorAndPlayer.com', 'user5EditorAndPlayer');
-    await users.login('user5@editorAndPlayer.com');
+    await users.createAndLoginUser(
+      'user5@editorAndPlayer.com', 'user5EditorAndPlayer');
 
     await workflow.createExploration();
     await explorationEditorMainTab.setStateName('card1');
```

Comment:
Below, we use the browser's back button to change pages without waiting for the page to load